modules/boost.uuid/1.87.0/MODULE.bazel

```diff
@@ -0,0 +1,23 @@
+module(
+    name = "boost.uuid",
+    version = "1.87.0",
+    bazel_compatibility = [">=7.2.1"],
+    compatibility_level = 108700,
+)
+
+
```

Comment:
Can you run `buildifier -r .` on this file - I guess it will remove one new line here... - maybe I am wrong...

Reply:
good eye, indeed